.circleci/config.yml

```diff
@@ -540,6 +540,7 @@ jobs:
     environment:
       TERM: xterm
       CMAKE_BUILD_TYPE: Release
+      MAKEFLAGS: -j 5
```

Comment:
I am not sure this is actually being picked up - it is at least not shown in the log.